rosbridge_library/src/rosbridge_library/capabilities/publish.py

```diff
@@ -56,6 +59,20 @@ def publish(self, message):
         latch = message.get("latch", False)
         queue_size = message.get("queue_size", 100)
 
+        if self.topics_glob:
```

Comment:
I think technically you want ```Publish.topics_glob``` here (and ```ClassName.foo_glob``` in other places).
```self.topics_glob``` implicitly copies the value of ```Publish.topics_glob``` the first time you access it, but you can set it independently afterwards.